scripts/dataset/main.js

```diff
@@ -0,0 +1,26 @@
+import logger from './logger/index.js';
+
+import { release } from './index.js';
+
+const RELEASE_HOURS_INTERVAL = 24;
+
+(() => {
+  const args = process.argv.slice(2);
+  const shouldPublish = args.includes('--publish');
+  const shouldRemoveLocalCopy = args.includes('--remove-local-copy');
+  const shouldSchedule = args.includes('--schedule');
+  const argsWithoutOptions = args.filter(arg => !arg.startsWith('--'));
+  const [fileName] = argsWithoutOptions;
+
+  if (!shouldSchedule) {
+    return release({ shouldPublish, shouldRemoveLocalCopy, fileName });
+  }
+
+  logger.info(`A release will be published every ${RELEASE_HOURS_INTERVAL} hours\n`);
```

Comment:
This is not entirely true: if `shouldPublish` is `false` but `shouldSchedule` is `true`, I understand that a local archive will be created every 24 hours but nothing will be published 🤪

I suggest that `shouldSchedule` implies `shouldPublish`: either override it to `true` after line 18, or `throw` if `shouldSchedule && !shouldPublish`. Another (lazier) option is to just remove this logging instruction and have a release silently created at the given interval, leaving it to the user to not ask for stupid things.

Reply:
IMO you can want to save the realeas file but not publish it anywhere and mount an ftp endpoint. So it makes sense to have both.

Reply:
True! But then the log is misleading :)